legato.gemspec

```diff
@@ -24,7 +24,7 @@ Gem::Specification.new do |s|
   # specify any dependencies here; for example:
   s.add_development_dependency "rake"
   s.add_development_dependency "rspec"
-  s.add_development_dependency "mocha"
+  s.add_development_dependency "mocha", '~> 1.9.0'
```

Comment:
I need to make an issue to get rid of mocha. rspec has enough mocking support to do a sufficient job. This is fine for now.